platforms/hyperledger-indy/images/README.md

```diff
@@ -15,10 +15,10 @@ Docker image contains Indy CLI, which is used to issue transactions agains an In
 For more information, see [Documentation](./indy-cli/README.md)
 
 ### indy-key-mgmt
-Docker image for indy key management, which allows using commands in bash.
+Docker image for indy key management, which generates identity crypto generation into Vault or on terminal output a in json format.
```

Comment:
Docker image for indy key management, which generates identity crypto and stores it into Vault or displays it onto the terminal in json format.